src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -352,7 +353,7 @@ paths:
       description: Returns the Temporary Certificate summaries for a Voter Card Application
       parameters:
         - name: applicationId
-          description: The identifier of the Voter Card Application.
+          description: The identifier of the Voter Card Application as a `query param`.
```

Comment:
I don't think " as a `query param`" is needed in the description - the schema makes that very clear with the `in: query`

Reply:
I don't think it does any harm, especially as `applicationId` is in the path of the vast majority of our endpoints.